docs/classic-ui/images.md

```diff
@@ -285,18 +285,34 @@ To access image scales, which are normally not accessible to the current user, o
 
 ## Responsive image support
 
-Plone supports the generation of picture tags with `srcset`s for image optimization.
+Plone supports the generation of [`picture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) tags with `srcset`s for image optimization.
 Additionally, you can define [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries) for [art direction](classic-ui-images-responsive-image-support-art-direction) and further optimization.
 
-The configuration allows you to define different picture variants, such as `Large`, `Medium`, or `Small`.
+The configuration allows you to define different `picture` variants, such as `Large`, `Medium`, or `Small`.
 Users can choose from them in editors, such as TinyMCE, and developers can use them in templates.
 
+To generate a `picture` tag, use the following code.
+
+```python
+from plone import api
+
+scale_util = api.content.get_view("images", context, request)
+tag = scale_util.picture("image", scale="larger", picture_variant="large")
+```
+
+The same can be done from a template;
+
+```xml
+<div tal:define="scale_view context/@@images">
+  <img tal:replace="structure python:scale_view.picture('image', scale='larger', picture_variant='large'" />
+</div>
```

Comment:
What about this version that does that without defining the scale view in a parent element?
```suggestion
<tal:image define="
             scale_view context/@@images;
           "
           replace="structure python:scale_view.picture('image', scale='larger', picture_variant='large'"
/>
```

Reply:
`tal:picture` if the result is a `<picture>` tag? just cosmetic, nothing important.

Reply:
This comment appears to be resolved, so I will click that button now, so we can merge.